spec/katello_certs_check_spec.rb

```diff
@@ -52,4 +52,16 @@ def fixture(filename)
       expect(status.exitstatus).to eq 4
     end
   end
+
+  context 'with wildcard certificate' do
+    let(:key) { File.join(certs_directory, 'wildcard.key') }
+    let(:cert) { File.join(certs_directory, 'wildcard.crt') }
+
+    it 'completes correctly' do
+      command_with_certs = "#{command} -b #{ca} -k #{key} -c #{cert}"
+      _stdout, stderr, status = Open3.capture3(command_with_certs)
```

Comment:
Does it make sense to also check `stdout` to make sure `Checking CA bundle size: ` with all the extras is on a single line?